components/Workspaces/TrackedRepoWizard/PasteReposStep.tsx

```diff
@@ -12,7 +12,14 @@
     // split each line into a trimmed string and filter out any empty lines
     const repos = pastedInput
       .split(pastedInput.includes(",") ? "," : "\n")
-      .map((line) => line.trim())
+      .map((line) => {
+        line.trim();
+        if (line.startsWith("https://github.com")) {
```

Comment:
I think in this particular case, we're not actually using the URL per se, so the worst that could happen is someone pastes an invalid repo, even if they're trying to be malicious. I think it's fine to dismiss this alert. Thoughts @jpmcb @brandonroberts?

Reply:
Probably negligable since we send it to the API and it does abit of its own validation. Do we attempt to use those URLs in any way to hit github? If so, we may want to attempt to escape malicious looking urls that attempt to sideload to other websites. Playing with it now to take a look.

Reply:
We currently don’t hit GH aside from loading an avatar URL with the org/repo-name